build.gradle.kts

```diff
@@ -3,8 +3,8 @@ import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
 import org.jetbrains.kotlin.gradle.targets.js.KotlinJsTarget
 import org.jetbrains.kotlin.konan.target.HostManager
 
-val coroutinesVersion = "1.5.2"
-val atomicfuVersion = "0.16.3"
+val coroutinesVersion = "1.6.0"
+val atomicfuVersion = "0.17.0"
 
 plugins {
     kotlin("multiplatform") version "1.6.0"
```

Comment:
did you mean to update to Kotlin 1.6.10?

Reply:
Yeah.